test/plot-schema.json

```diff
@@ -2074,7 +2074,55 @@
        "albers usa",
        "winkel tripel",
        "aitoff",
-       "sinusoidal"
+       "sinusoidal",
+       "airy",
+       "august",
+       "boggs",
+       "bromley",
+       "collignon",
+       "craster",
+       "cylindrical equal area",
+       "cylindrical stereographic",
+       "eckert1",
+       "eckert2",
+       "eckert3",
+       "eckert5",
+       "eckert6",
+       "eisenlohr",
+       "fahey",
+       "foucaut",
+       "foucaut sinusoidal",
+       "ginzburg5",
+       "ginzburg6",
+       "ginzburg8",
+       "ginzburg9",
+       "gringorten",
+       "guyou",
+       "hill",
+       "hufnagel",
+       "hyperelliptical",
+       "lagrange",
+       "mt flat polar parabolic",
+       "mt flat polar quartic",
+       "mt flat polar sinusoidal",
+       "natural earth1",
+       "natural earth2",
+       "nell hammer",
+       "nicolosi",
+       "patterson",
+       "rectangular polyconic",
+       "satellite",
+       "times",
+       "van der grinten",
+       "van der grinten2",
+       "van der grinten3",
+       "van der grinten4",
+       "wagner4",
+       "wagner6",
+       "wiechel",
+       "winkel3",
+       "gringorten quincuncial",
+       "peirce quincuncial"
```

Comment:
Is this why you were asking about sorting? I still don't think we want to sort at the schema level, but we may want to sort these values in the attribute spec - ie add sorting directly to [`Object.keys(constants.projNames)`](https://github.com/plotly/plotly.js/blob/ec01107b5fa289ded60b858c5f61741c2d4e19bf/src/plots/geo/layout_attributes.js#L110) - and perhaps anywhere else we use `Object.keys` to get enum values for attributes.

Reply:
Projection values are sorted in https://github.com/plotly/plotly.js/pull/5801/commits/5fd34286a17080e8fbfc578191fdb8da2dfd1e78.

I'd look for other Object.keys `values` and possibly address that outside this PR.

Reply:
> I'd look for other Object.keys `values` and possibly address that outside this PR.

Addressed in #5813.